example-copilot/example_copilot/models.py

```diff
@@ -1,4 +1,5 @@
 from typing import Any
+from uuid import UUID
 from pydantic import BaseModel, Field, validator
```

Comment:
"Pydantic V1 style `@validator` validators are deprecated. You should migrate to Pydantic V2 style `@field_validator` validators, see the migration guide for more details"

Should be just swapping `validator` to `field_validator`

Reply:
That's a great point, and a good catch. Thank you! Will update.